tools/coverage_plugins_package/src/coverage_plugins/jit_plugin.py

```diff
@@ -41,9 +41,17 @@ def dynamic_context(self, frame):
                 filename = getsourcefile(obj)
                 # We don't want to report for filename = None
                 if filename:
-                    sourcelines, starting_lineno = getsourcelines(obj)
-                    line_data = {filename: range(starting_lineno, starting_lineno + len(sourcelines))}
-                    cov_data.add_lines(line_data)
+                    # TODO: Because torch.jit._IgnoreContextManager relies on
+                    # python `exec` method that doesn't generate source codelines
+                    # , this part of code fails. Therefore, we just ignore the exception
+                    # until we figure out a better way to implement torch.jit._IgnoreContextManager.
```

Comment:
```suggestion
                    # TODO: Because torch.jit._IgnoreContextManager relies on Python's `exec` method which doesn't 
                    # generate source codelines, getsourcelines(obj) fails. For now, we just ignore the exception
                    # until we figure out a better way to implement torch.jit._IgnoreContextManager.
```
for more clarity